specifyweb/frontend/js_src/lib/components/Molecules/ResourceLink.tsx

```diff
@@ -72,7 +73,7 @@ export function ResourceLink<COMPONENT extends typeof Link['Icon']>({
       <AnyComponent {...allProps} />
       {isOpen && (
         <IsNotReadOnly.Provider value>
-          <ResourceView
+          <LazyResourceView
```

Comment:
Currently, this is the only use of LazyResourceView. 
Can the function be used anywhere else to improve performance or was this the only place where it can be used or was a significant bottleneck?

Reply:
in other places where ResourceView is used, most of ResourceView's dependencies would have to be loaded anyway.

i.e if ResourceView is called from inside of ResoruceView, then ResourceView is already loaded, and so using LazyResourceView in that place would add (tiny) overhead.

I guess the query "View in forms" is one place where this might help a bit, but probably won't be noticeable.

---

on the other hand, the ResourceLink was called from the Notifications rendered, which is from the Header, which is called from everywhere - meaning every single webpage in specify was loading ResourceView and all of it's dependencies (attachments, all form cells and plugins, etc...)
and so all that extra code on the home page adds extra time to 1. network fetching (though not a big issue for an app like specify as js bundles are going to be cached, and aren't updated often) 2. browser needs time to parse all that JS (100ms as seen in my benchmarking - not too much, but that's 100ms on every page load for every user, so it adds up), 3. more ram usage since more functions are declared/objects created needlessly (this one might be more significant for performance on weaker systems)